test/integration/standalone/standalone_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package standalone
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func getPolicyID(t *testing.T, orgID string) string {
+	gcOpts := gcloud.WithCommonArgs([]string{"--format", "value(name)"})
+	op := gcloud.Run(t, fmt.Sprintf("access-context-manager policies list --organization=%s ", orgID), gcOpts)
+	return op.String()
+}
+
+func TestStandalone(t *testing.T) {
+	orgID := utils.ValFromEnv(t, "TF_VAR_org_id")
+	policyID := getPolicyID(t, orgID)
+
+	for _, tt := range []struct {
+		name string
+	}{
+		{
+			name: "standalone",
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+
+			vars := map[string]interface{}{
+				"access_context_manager_policy_id": policyID,
+			}
+
+			standalone := tft.NewTFBlueprintTest(t,
+				tft.WithVars(vars),
+			)
+
+			standalone.DefineVerify(func(assert *assert.Assertions) {
+				standalone.DefaultVerify(assert)
+
+				gcloudArgs := gcloud.WithCommonArgs([]string{"--format", "json"})
+
+				dataGovprojectID := standalone.GetStringOutput("data_governance_project_id")
+				opdataGov := gcloud.Run(t, fmt.Sprintf("projects describe %s", dataGovprojectID), gcloudArgs)
```

Comment:
https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud#Runf here and throughout to make it simpler
```suggestion
				opdataGov := gcloud.Runf(t, "projects describe %s", dataGovprojectID)
```